sass/site/primary/_comments.scss

```diff
@@ -217,7 +217,10 @@
 			height: 18px;
 			position: absolute;
 			background: lighten( $color__link, 8% );
-			right: calc(100% + #{$size__spacing-unit * .25});
+			right: calc(100% - #{$size__spacing-unit * 3});
+			@include media(tablet) {
```

Comment:
This looks good but could you move the media query mixin chunk to its own line below this group of rules? It should probably start at line `227`.

See how the media query mixin appears in other places for examples. Thanks!

Reply:
Yeah. Done.
Please check it's right.